server-app/src/persistence/project_DAO.py

```diff
@@ -1,13 +1,12 @@
 from .project import Project
+from bson import ObjectId
 
 
 def createProject(projectJson):
     new_project = Project(**projectJson)
     new_project.commit()
-    # project = projectJson.dump()
-    # created_project = Project.find_one({"projectId": project.projectId})
 
-    return 1
+    return Project.find_one({"projectId": new_project.projectId}).dump()
```

Comment:
if the project insert was successful, wouldn't it be enough to return the `new_project` instance?